### DISCUSSION TITLE
Migrate classic sidebars and widgets to block for themers

### DISCUSSION THREAD

**Discussion by justintadlock** [WordPress/developer-blog-content#206] [Topic Ideas]
We could use a good general guide that covers sidebars and widgets. Widgets are a feature that both themes and plugins created during the classic era. However, blocks are more plugin territory. My focus for this article would be on looking at this from a themer's viewpoint:

- Dynamic sidebars (i.e., widget areas) should be replaced with template parts, patterns, etc. No more registration; anything can go anywhere in block templates!
- Moving a custom widget to a block. I'm thinking of this more in broad strokes (every widget has different options and different uses).

This discussion was branched off an earlier one that is specific to plugins and more broad in scope: https://github.com/WordPress/developer-blog-content/discussions/42

**Comment by bph** [comment]
This topic was approved, the discussion was locked and conversation moved to the issue 
- #216
